src/forms/Checkbox.jsx

```diff
@@ -1,11 +1,12 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { Checkbox as SwarmCheckbox } from '@meetup/swarm-components';
+import DeprecationWarning from '../utils/components/DeprecationWarning';
 
 /**
  * @module Checkbox
  */
-class Checkbox extends React.PureComponent {
+export class Checkbox extends React.PureComponent {
```

Comment:
Does this mean that if somebody imports Checkbox explicitly, they will not get the warning?
If we don't have any code that does that, why do we need to export it now?

Reply:
It's for tests. The higher order component now wraps the default export causing the tests to not have the proper instance without it